docs/UserGuide.md

```diff
@@ -302,8 +302,8 @@ _Details coming soon ..._
 
 ### Date of Last Visit
 * Dates of last visit are optional for contacts, and are denoted by the `d/` prefix.
-* Dates of last visit are confined to the `DD-MM-YYYY` format.
-* The date provided must be valid, and before the current date. This prevents accidental entering of future dates.
+* Dates of last visit are confined to the `DD-MM-YYYY` format and follows the ISO-8601 calendar system.
```

Comment:
A random user might not know what ISO-8601 is. One option maybe to add a link to clarify. Just something to consider because hard terms can count as a bug.